### ISSUE TITLE
move to webkitgtk4

### ISSUE THREAD

**Issue by quozl** [sugarlabs/sugar-build#50]
Downstream Fedora package for Sugar has moved from webkitgtk3 to webkitgtk4, see http://pkgs.fedoraproject.org/cgit/rpms/sugar.git/commit/?id=6e4b6d849795bb1b129a507d954636773ce424e7

`sugar-build` uses Fedora packages, and will have to catch up with this change which happened in upstream `sugar`.

**Comment by Himanshu40** [comment]
I am on it but it showing so many errors while updating `sugar-build` for Fedora 25. Can you show me which packages are needed and which are not????

**Comment by quozl** [comment]
No, I can't show you that, as I don't use `sugar-build` nor have I done the port to Fedora 25.  If I had, I'd commit the changes immediately.

Every single package should be there for a reason; and you should be able to deduce the reason by looking for references to them in the Sugar, Toolkit, Artwork, Datastore, or activity source code.

Fedora 23 is different to Fedora 25, in that the mix of packages and versions is different; and that's the whole point of a Fedora 25 release; to add features and fix bugs by updating versions.  Sometimes that will break something apart, and you get to keep both pieces and find out why.

You might look at the [history of changes to the root.json file](https://github.com/sugarlabs/sugar-build/commits/master/build/root.json); you'll know you are on the right track once you have internalised and understood the reason for each of those commits.

Porting `sugar-build` to a new version of Fedora is probably _the most advanced_ technical task you can do in Sugar Labs. :grin: Thanks for offering to help.
